components/quick_switch_modal/quick_switch_modal.jsx

```diff
@@ -111,7 +111,7 @@ export default class QuickSwitchModal extends React.PureComponent {
 
         if (this.state.mode === CHANNEL_MODE) {
             const selectedChannel = selected.channel;
-            this.props.actions.switchToChannelById(selectedChannel.id).then((result) => {
+            this.props.actions.switchToChannel(selectedChannel).then((result) => {
```

Comment:
This is the fix for MM-13153

Reply:
Curious to know why was this changed to `switchToChannelById` after we changed this to `switchToChannel` when we did the migration to redux? I only ask cause is being reverted once more to be `switchToChannel`

Reply:
Oh looks like this did it to fix GMs https://github.com/mattermost/mattermost-webapp/pull/2047/files

I should check that case and fix it if need be

Reply:
sounds good, add me to re-review it if needed